pkg/reconciler/autoscaling/kpa/scaler_test.go

```diff
@@ -186,6 +212,21 @@ func TestScaler(t *testing.T) {
 			markSKSInProxyFor(s, gracePeriod-time.Second)
 		},
 		wantCBCount: 1,
+	}, {
+		label:         "waits to scale to zero (just before grace period, sks in proxy long) and last pod timeout positive",
+		startReplicas: 1,
+		scaleTo:       0,
+		wantReplicas:  0,
+		wantScaling:   true,
+		paMutation: func(k *pav1alpha1.PodAutoscaler) {
+			paMarkInactive(k, time.Now().Add(-gracePeriod).Add(time.Second))
+		},
+		configMutator: func(c *config.Config) {
+			c.Autoscaler.ScaleToZeroPodRetentionPeriod = 42 * time.Second
```

Comment:
Could you add a comment that 42 * time.Second is less than gracePeriod, which is 60 * time.Second in test.

Reply:
Sure.